scripts/prepare-testing-project.sh

```diff
@@ -38,4 +38,4 @@ echo "Creating pubsub resources."
 gcloud alpha pubsub topics create gae-mvm-pubsub-topic
 
 echo "To finish setup, follow this link to enable APIs."
-echo "https://console.cloud.google.com/flows/enableapi?apiid=datastore,pubsub,storage_api,logging,plus,bigquery,cloudmonitoring,compute_component"
+echo "https://console.cloud.google.com/flows/enableapi?project=${GCLOUD_PROJECT}&apiid=bigquery,cloudmonitoring,compute_component,datastore,dataproc,dns,plus,pubsub,logging,storage_api,vision.googleapis.com"
```

Comment:
Done in latest commit.

I don't remember having to enable that API. It could be on by default, but explicit is better than implicit.